src/app/[locale]/_component/accordion/faqDefault.tsx

```diff
@@ -73,8 +77,47 @@ const fifthBlockFaqRick = {
     </Link>
   ),
 };
+const fifthRevokeWalletFaqRick = {
+  Link: (chunks: React.ReactNode) => {
+    const { locale } = useParams()
+    return <Link
+      fontWeight={600}
+      href={`/${locale}${ROUTES.PROFILE_BLOCK}`}
```

Comment:
it looks like you can use ${baseUrl}/${localeFromStorage} instead of useParams() to get the locale from the URL. This way you can avoid the issue with the locale being undefined when the component is first rendered. See previous Link component for an example.